### PR TITLE
Docs: add ranking question documentation

### PR DESCRIPTION
**Pull request by nataliaElv**
# Description

Documentation changes after the addition of the ranking question

**Type of change**
- [x] Documentation update

**Checklist**

- [ ] I added relevant documentation
- [ ] follows the style guidelines of this project
- [ ] I did a self-review of my code
- [x] I made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I filled out [the contributor form](https://tally.so/r/n9XrxK) (see text above)
- [ ] I have added relevant notes to the CHANGELOG.md file (See https://keepachangelog.com/)

### CODE REVIEW

File: docs/_source/guides/llms/practical_guides/collect_responses.md [hunk 1]
```diff
@@ -81,33 +81,35 @@ If you feel that the disagreement between annotators is too high, especially for
...
 * *Weighted majority vote*: You may decide to give some of your annotators more weight than others when aggregating labels so that their decisions count more than others. Some reasons to consider a weighted majority might be: because some annotators tend to have better agreement with ground truth annotations, they are experts or they represent the demographic target for specific questions. If you want to choose this option, first calculate a score from 0 to 1 for each annotator, then apply these weights to their responses. Finally, sum all the values and choose the option with the highest score.
 * *Train with disagreements*: If your labels are meant to solve highly subjective tasks, like sentiment analysis or abusive language detection, you may want to consider other options that preserve the natural disagreement between annotators during training. This is specially helpful to avoid diluting the feedback of minorities within your annotation team. If you want to learn more about this approach, we recommend checking the different methods discussed in [Davani et al. (2021)](https://arxiv.org/pdf/2110.05719.pdf).
+
+#### For numerical values: `RankingQuestion` and `RatingQuestion`
```
Comment (alvarobartt) [inline docs/_source/guides/llms/practical_guides/collect_responses.md:104]: I think @davidberenstein1957 mentioned that the support for `RankingQuestion` is not ready yet, but he'll work on it for the next release, maybe we can either skip it or just add a warning that support via Python client is not ready yet?

**Comment by alvarobartt** [comment]
Hi @nataliaElv great work!

I think we also need to remove `(upcoming)` from https://github.com/argilla-io/argilla/blob/develop/docs/_source/guides/llms/conceptual_guides/data_model.md#question, in the `RankingQuestion` bullet point 😄

**Comment by alvarobartt** [comment]
And also in https://github.com/argilla-io/argilla/blob/develop/docs/_source/guides/llms/conceptual_guides/rm.md, we need to remove the references to the future addition of `RankingQuestion` as it's already here 👍🏻

File: docs/_source/guides/llms/practical_guides/collect_responses.md (hunk 1 above)
Comment (nataliaElv) [reply docs/_source/guides/llms/practical_guides/collect_responses.md:104]: There are other approaches in that section that we don't support in the Python client and there's a disclaimer before the code section saying that we only cover some, so I think it should be ok to keep it there.

**Review by alvarobartt — APPROVED** [review]
LGTM, feel free to merge!

**Pull request merged**